libraries/AP_Mount/AP_Mount.cpp

```diff
@@ -478,9 +497,25 @@ void AP_Mount::init(const AP_SerialManager& serial_manager)
 // update - give mount opportunity to update servos.  should be called at 10hz or higher
 void AP_Mount::update()
 {
+    AP_AHRS &ahrs = AP::ahrs();
+
     // update each instance
     for (uint8_t instance=0; instance<AP_MOUNT_MAX_INSTANCES; instance++) {
         if (_backends[instance] != nullptr) {
+            if (state[instance]._retract_altitude != 0 && ahrs.get_likely_flying() && get_mode() != MAV_MOUNT_MODE_RETRACT) {
+                float alt;
+                ahrs.get_relative_position_D_home(alt);
+                if (-alt < state[instance]._retract_altitude) {
+                    if (_reached_altitude){
+                        _backends[instance]->set_mode(MAV_MOUNT_MODE_RETRACT);
+                        gcs().send_text(MAV_SEVERITY_INFO, "Low altitude %d. Retracting mount.", (int)alt);
```

Comment:
Personally I don't think the message is useful and I'd rather we don't send it - wait for other opinions before you remove it though.